common-service-core/src/test/java/com/ca/mfaas/message/core/AbstractMessageServiceTest.java

```diff
@@ -69,6 +71,19 @@ public void shouldCreateMessages_IfMultipleParametersArePassed() {
         assertEquals("Message texts are different", "No response received within the allowed time: 3000", messages.get(1).getConvertedText());
     }
 
+    @Test
+    public void shouldNotCreateMessages_IfEmptyParametersArePassed() {
+        List<Object[]> parameters = new ArrayList<>();
+        List<Message> messages = abstractMessageService.createMessage("apiml.common.serviceTimeout", parameters);
+        assertEquals("Generated different number of messages than expected", 0, messages.size());
+    }
+
+    @Test
+    public void shouldNotCreateMessages_IfEmptyParameterListIsPassed() {
+        Message messages = abstractMessageService.createMessage("apiml.common.stringParamMessage", new ArrayList<String>());
+        assertEquals("Generated different number of messages than expected", "This message has one param: []", messages.getConvertedText());
```

Comment:
I think you forgot to change test message "Generated different number of messages than expected". It is the same message with the previous one.

Reply:
Thanks, you got me : )